tox.ini

```diff
@@ -36,45 +58,38 @@ passenv =
     PASS_INVALID_VALUES
     VALIDATE_ON_ASSIGNMENT
     TEST_BIGDATA
-
 usedevelop = true
-
 commands =
-    !cov: pytest -n auto {posargs}
-    cov: pytest -n auto --cov-report=xml --cov=. --cov-config=setup.cfg {posargs}
-
+    pip freeze
+    pytest -n auto \
```

Comment:
Generally using `pytest -n auto` I don't think is desirable.  This is only useful for running regtests, which actually doesn't run on Github actions but runs under Jenkins, where there are dozens of cores available.  Github actions only has a couple cores available.

Reply:
Ah, I understand now.  That's clever.  Carry on.  🚀